src/Symfony/Bundle/FrameworkBundle/Resources/config/rate_limiter.php

```diff
@@ -11,15 +11,15 @@
 
 namespace Symfony\Component\DependencyInjection\Loader\Configurator;
 
-use Symfony\Component\RateLimiter\RateLimiter;
+use Symfony\Component\RateLimiter\RateLimiterFactory;
 
 return static function (ContainerConfigurator $container) {
     $container->services()
         ->set('cache.rate_limiter')
             ->parent('cache.app')
             ->tag('cache.pool')
 
-        ->set('limiter', RateLimiter::class)
+        ->set('limiter', RateLimiterFactory::class)
```

Comment:
Does it till make sens to name it `limiter`?

Reply:
Thank you. I renamed the service to `limiter_factory`. 

I did not use `limiter.factory` as it may conflict with a service dynamically defined from config.